README.md

```diff
@@ -1,11 +1,11 @@
 # HPE OneView SDK for GoLang
 
 ## Build Status 
-
 OV Version | 6.00 | 5.60 | 5.50 | 5.40 | 5.30 |
-| ------------- |:-------------:|-------------:|-------------:|-------------:|
-SDK Version/Tag | [v6.0.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.0.0) |[v1.8.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.8.0) |[v1.7.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.7.0) | [v1.6.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.6.0) | [v1.5.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.5.0) |
-Build Status | ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true) | ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true) | ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true) | ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)|
+| ------------- |:-------------:| -------------:| -------------:| -------------:| -------------:|
+SDK Version/Tag | [v6.0.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v6.0.0) | [v1.8.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.8.0) | [v1.7.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.7.0) | [v1.6.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.6.0) | [v1.5.0](https://github.com/HewlettPackard/oneview-golang/releases/tag/v1.5.0) | 
+Build Status | ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)|
```

Comment:
Please attach proper build logs url to the build badge the same way we made it in ansible repo.
Please refer this https://github.com/HewlettPackard/oneview-ansible/blob/master/README.md